lib/node_modules/@stdlib/array/bool/README.md

```diff
@@ -779,6 +779,106 @@ var count = context.count;
 // returns 3;
 ```
 
+<a name="method-reduce"></a>
+
+#### BooleanArray.prototype.reduce( reducerFn\[, initialValue] )
+
+Applies a provided callback function to each element of the array, in order, passing in the return value from the calculation on the preceding element and returning the accumulated result upon completion.
+
+```javascript
+var Boolean = require( '@stdlib/boolean/ctor' );
+
+function reducer( acc, v ) {
+    return Boolean( acc & v );
+}
```

Comment:
Can we remove Boolean constructor from here, by return `acc && v` ?

Reply:
You don't need the boolean constructor and can remove.